jupyter_scheduler/environments.py

```diff
@@ -37,10 +44,7 @@ def list_environments(self) -> List[RuntimeEnvironment]:
                     label=name,
                     description=f"Conda environment: {name}",
                     file_extensions=["ipynb", "py"],
```

Comment:
Does our default executor actually support `.py`. files? If so, can we wire up stdout and stderr as output formats?

Reply:
I don't think it will work, I will remove `py` from input here.